README.md

```diff
@@ -1,2 +1,131 @@
-# termviz
-ROS visualisation on the terminal
+# TermViz - ROS visualization on the terminal
+
+TermViz is a visualizer for the Robot Operating System (ROS) framework, that runs completely on the terminal.
+This allows the visualizer to run directly on the robot (e.g., via ssh) without any configuration on an external computer.
+It can also be useful in case the robot has a firewall or in situations in which no graphics server can be run.
+
+## How to use
+
+To launch the visualizer, just run the `termviz` executable.
+
+The program looks for a configuration file named `termviz.yml` in `~/.config/termviz/` first, then in `/etc/termviz/`. If the file is not found, it prompts the user to create a default one in `~/.config/termviz/termviz.yml`. Alternatively, it is possible to pass a configuration file directly to the executable: `termviz <myconfig>.yml`.
```

Comment:
remove termviz.yml in the  last path or rephrase:
it creates the configuration: ....

Reply:
I don't understand this comment. What do you mean exactly?